### PR TITLE
update conda to version 4.3.21

### LINKED ISSUE #16: conda 4.3?
I'm actually puzzled about an independent conda-feedstock, shouldn't all conda repos best be managed by the same conda executable? A different channel hopefully doesn't depend on special features of any specific `conda` version?
I'm usually basing my env on conda-forge due to some updates on packages I frequently use, is there any danger in using conda 4.3 from the defaults channel with the current packages of the conda-forge channel?

### PR DESCRIPTION
**Pull request by slavoutich**
Supersedes #21, fixes #16.

### CODE REVIEW

**Comment by slavoutich** [comment]
(Is there another way to run CI except sending a pull request?)

File: ci_support/run_docker_build.sh [hunk 1]
```diff
@@ -24,12 +24,14 @@ show_channel_urls: true
...
 
+rm -f "$FEEDSTOCK_ROOT/build_artefacts/conda-forge-build-done"
+
 cat << EOF | docker run -i \
                         -v "${RECIPE_ROOT}":/recipe_root \
                         -v "${FEEDSTOCK_ROOT}":/feedstock_root \
                         -a stdin -a stdout -a stderr \
                         condaforge/linux-anvil \
-                        bash || exit $?
+                        bash || exit 1
```
Comment (scopatz) [inline ci_support/run_docker_build.sh]: Isn't `exit $?` more correct here, so the error code is known?

Comment (scopatz) [inline ci_support/run_docker_build.sh]: Or was this just part of a rerender?

Comment (slavoutich) [inline ci_support/run_docker_build.sh]: @basnijholt Any reason why have you done this?

Comment (scopatz) [inline ci_support/run_docker_build.sh]: It does look like this was part of a rerender

Comment (slavoutich) [inline ci_support/run_docker_build.sh]: (I believe your conjecture is right, though, as follows from the commit message.)

**Comment by scopatz** [comment]
Thanks!

**Comment by slavoutich** [comment]
Hurray! :)

**Comment by jakirkham** [comment]
So this is fun. Looks like releasing `conda` itself on Windows is breaking because `conda` wants to downgrade from Python 3 to Python 2 to get the latest version of itself. FYI this is actually breaking other Windows builds as well. 😒 Will submit a hack to the feedstock to fix this.

ref: https://ci.appveyor.com/project/conda-forge/conda-feedstock/branch/master/job/e2f4vdvbk19obcg0

**Comment by scopatz** [comment]
Uggh, thanks for tracking this down @jakirkham!

**Pull request merged**